static-analysis/src/main/java/com/example/static_analysis/LichIssueRegistry.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.static_analysis
```

Comment:
Please change the package name to `com.linecorp.lich.static_analysis`.

Reply:
Right, I missed this, thanks!